SCM-Overhaul/common/economic_categories/00_common_categories.txt

```diff
@@ -1163,7 +1163,6 @@ claims = {
 }
 
 subject_tax = {
-	parent = diplomacy
```

Comment:
Idk if this will have unintended consequences, but after this change fanatic xenophobes had the normal 25% tribute tax, so did xenophiles and those having the diplomacy opener.